ocfweb/docs/docs/services/web.md

```diff
@@ -11,6 +11,49 @@ where `user` is the account name.
 Group accounts may also request a [[virtual host|doc services/vhost]] for
 another domain (e.g., group.berkeley.edu).
 
+#### Off-site hosting
+
+One question that often comes up is if the OCF is able to host websites
+from services such as Squarespace, Wix, Weebly, Tumblr, etc. Unfortunately
+due to logistical considerations and University policy, the OCF is unable
+to do so.
+
+First, many website building services like Wix and Weebly make their money
+by using a proprietary service to allow users to build their sites, and
+subsequently charging users to host those sites. This means providing the
+servers, storage space, internet connection, and configurations to allow
+these sites to show up on the internet. Obviously, the OCF is unable to
+access these proprietary backends and is therefore unable to host sites
```

Comment:
Using the word `backend` will not make a lot of sense to most people wanting to host with us, maybe just use something like `access this proprietary software` instead?